packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -503,7 +503,7 @@ class Neo4jGraphQL {
 
         const setup = async () => {
             const subscriptionsEngine = this.features?.subscriptionsEngine;
-            if (subscriptionsEngine) {
+            if (subscriptionsEngine && (await this.executableSchema)?.getSubscriptionType()) {
```

Comment:
not too happy about having an await as part of a logical expression, maybe move it into a separate variable?